components/observatorium.libsonnet

```diff
@@ -92,6 +93,9 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           },
           replicas: 1,
           ignoreDeletionMarksDelay: '24h',
+          memcached+: {
+            addresses: ['dnssrv+_http._tcp.%s.%s.svc.cluster.local' % [obs.storeCache.service.metadata.name, obs.storeCache.service.metadata.namespace]],
```

Comment:
we can't rely on the `.cluster.local` suffix

Reply:
@brancz  Could you please explain a little bit more? We already rely heavily on `.cluster.local` across the board, what do I miss?

Reply:
We do :scream: . The problem is that this suffix is configurable for every kubernetes, cluster so we need to omit it. The local DNS server will still resolve it correctly for the local service, but it may not be correct if we add the suffix.

Reply:
tl;dr We need to remove it whereever we use it.